packages/app-project/src/shared/components/AboutProject/AboutProject.js

```diff
@@ -0,0 +1,34 @@
+import counterpart from 'counterpart'
+import { Paragraph } from 'grommet'
+import PropTypes from 'prop-types'
```

Comment:
Please destructure the required `propTypes`: 

```
import { string } from 'prop-types'

...

AboutProject.propTypes = {
  projectName: string,
  description: string
}
```

Reply:
I updated this file, as well as the others that use Proptypes in this PR: https://github.com/zooniverse/front-end-monorepo/pull/660/commits/c8d8505241006145642d37357f2f201d3c74f783